project/app/(dashboard)/projects/[projectSlug]/[taskSlug]/page.tsx

```
@@ -60,163 +59,172 @@ export default function TaskDetails({ params }: TaskProps) {
-                        <span className="font-semibold">Due Date:</span>{" "}
-                        {task.dueDate
-                            ? new Date(task.dueDate).toLocaleDateString()
-                            : "None"}
+            <div className="doc-header flex flex-col justify-between gap-4">
+                <div className="flex flex-col gap-2">
+                    <div className="flex flex-row justify-between">
+                        <div className="flex flex-row items-center gap-4">
+                            <TypographyH1>{task.title}</TypographyH1>
+                            <TypographyMuted>
+                                {" "}
+                                // {task.projectName}
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)